### ISSUE TITLE
不知道训练集放哪

### ISSUE THREAD

**Issue by Aruelius** [kerlomz/captcha_trainer#8]
你好，我在TensorFlow这是个新手，我按照您给的教程，TestSetNum设置为10w之后，运行了trains.py，我看到在进行到1500步的时候，停下来提示训练完成，然后我按照部署教程部署，post之后永远是结果8，换图片都是这样。
请问训练用的验证码图片放在哪？
我的操作是否有误。
![image](https://user-images.githubusercontent.com/25380989/55052257-37514b80-5093-11e9-8361-2f3ae035f7f5.png)
![image](https://user-images.githubusercontent.com/25380989/55052260-3ddfc300-5093-11e9-9bc5-cfe69ab5714e.png)
![image](https://user-images.githubusercontent.com/25380989/55052273-47692b00-5093-11e9-9697-fd3f8796d36b.png)

**Comment by kerlomz** [comment]
训练集的设置可以在config.yaml中的
`
Trains:
  TrainsPath: '训练集路径'
  TestPath: '测试集集路径（如果使用TestSetNum参数，并且非tfrecords集，可以不填）'
`
你描述的问题有几种可能：
1、部署的模型不是你训练的模型
2、你训练的时候训练集用错了，可能用了给你们入门配的mnist训练集
3、你训练过mnist模型之后，没有修改model_name直接进行下一轮训练，默认是继续上一个训练的
4、标签的正则没有匹配对，获取到的标注不正确

检查一下以上的可能性，希望对你有所帮助

**Comment by MikuShare** [comment]
谢谢，我这边再做尝试。

**Comment by dooozi** [comment]
@Aruelius 大佬 能否帮忙看下我的训练结果么？ 我看的不是很明白，也不清楚是否有问题。。

INFO:tensorflow:Step: 9200 Time: 13.979 sec/batch, Cost = 0.00025, 160x60-BatchSize: 128
INFO:tensorflow:Step: 9300 Time: 13.981 sec/batch, Cost = 0.00025, 160x60-BatchSize: 128
INFO:tensorflow:Step: 9400 Time: 13.941 sec/batch, Cost = 0.00025, 160x60-BatchSize: 128
INFO:tensorflow:Step: 9500 Time: 14.033 sec/batch, Cost = 0.00024, 160x60-BatchSize: 128
INFO:tensorflow:Step: 9600 Time: 14.009 sec/batch, Cost = 0.00024, 160x60-BatchSize: 128
INFO:tensorflow:Step: 9700 Time: 13.956 sec/batch, Cost = 0.00024, 160x60-BatchSize: 128
INFO:tensorflow:Step: 9800 Time: 14.029 sec/batch, Cost = 0.00023, 160x60-BatchSize: 128
INFO:tensorflow:Step: 9900 Time: 14.055 sec/batch, Cost = 0.00023, 160x60-BatchSize: 128
INFO:tensorflow:Step: 10000 Time: 14.234 sec/batch, Cost = 0.00023, 160x60-BatchSize: 128
INFO:tensorflow:0 7 7 [13, 11, 26, 30, 13, 18, 11] [13, 11, 26, 30, 13, 18, 11] --> ['c', 'a', 'p', 't', 'c', 'h', 'a'] ['c', 'a', 'p', 't', 'c', 'h', 'a']
INFO:tensorflow:1 7 7 [13, 11, 26, 30, 13, 18, 11] [13, 11, 26, 30, 13, 18, 11] --> ['c', 'a', 'p', 't', 'c', 'h', 'a'] ['c', 'a', 'p', 't', 'c', 'h', 'a']
INFO:tensorflow:2 7 7 [13, 11, 26, 30, 13, 18, 11] [13, 11, 26, 30, 13, 18, 11] --> ['c', 'a', 'p', 't', 'c', 'h', 'a'] ['c', 'a', 'p', 't', 'c', 'h', 'a']
INFO:tensorflow:3 7 7 [13, 11, 26, 30, 13, 18, 11] [13, 11, 26, 30, 13, 18, 11] --> ['c', 'a', 'p', 't', 'c', 'h', 'a'] ['c', 'a', 'p', 't', 'c', 'h', 'a']
INFO:tensorflow:4 7 7 [13, 11, 26, 30, 13, 18, 11] [13, 11, 26, 30, 13, 18, 11] --> ['c', 'a', 'p', 't', 'c', 'h', 'a'] ['c', 'a', 'p', 't', 'c', 'h', 'a']
ERROR:tensorflow:[]
INFO:tensorflow:Epoch: 1358, Step: 10000, Accuracy = 1.0000, Cost = 0.00023, Time = 15.367 sec/batch, LearningRate: 0.0009800000116229057
INFO:tensorflow:CNN Output: (?, ?, 256)
INFO:tensorflow:./model/myfirsttest.model-10000
INFO:tensorflow:Restoring parameters from ./model/myfirsttest.model-10000
INFO:tensorflow:Froze 40 variables.
Converted 40 variables to const ops.
INFO:tensorflow:Total Time: 33.93611812591553 sec.
INFO:tensorflow:Training completed.

**Comment by Aruelius** [comment]
@dooozi 建议重开Issues，我也不是太清楚。问问作者本人吧。
